File: blackjack/Card.php
<?php

class Card
{
    public $value;
    public $suit;

    public function __construct($value, $suit)
    {
        $this->value = $value;
        $this->suit = $suit;
    }

    public function toString()
    {
        return ('[' . $this->value . ' ' . $this->suit . ']');
    }

    public function getBlackjackValue()
    {
        if (is_numeric($this->value)) {
            return (int) $this->value;
        } elseif ($this->value == 'A') {
            return 11;
        } else {
            return 10;
        }
    }
}

File: blackjack/Card.rb
class Card
    def initialize value, suit
        @value = value
        @suit = suit
    end

    def toString
        "[#{@value} #{@suit}]"
    end

    def getBlackjackValue
        if  @value.to_i != 0
            return @value.to_i
        elsif @value == 'A'
            return 11
        else
            return 10
        end
    end

end
